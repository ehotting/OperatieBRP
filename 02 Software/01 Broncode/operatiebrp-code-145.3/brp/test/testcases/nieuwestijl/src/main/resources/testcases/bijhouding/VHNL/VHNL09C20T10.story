Meta:
@status                 Klaar
@usecase                UCS-BY.HG

Narrative:
Datumformaten. Punt als scheidingsteken van de datumcomponenten

Scenario:   Punt als scheidingsteken van de datumcomponenten
            LT: VHNL09C20T10

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL/Libby.xls

When voer een bijhouding uit VHNL09C20T10.xml namens partij 'Gemeente BRP 1'

Then is het foutief antwoordbericht gelijk aan /testcases/bijhouding/VHNL/expected/VHNL09C20T10.txt

Then is in de database de persoon met bsn 422531881 niet als PARTNER betrokken bij een HUWELIJK