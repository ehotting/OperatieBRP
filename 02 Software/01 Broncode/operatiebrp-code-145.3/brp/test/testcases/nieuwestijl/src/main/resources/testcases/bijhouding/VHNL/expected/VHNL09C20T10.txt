cvc-pattern-valid: Value '01.01.1980' is not facet-valid with respect to pattern '[0-9]{4}(-(0[1-9]{1}|1[0-2]{1})(-(0[1-9]{1}|[1-2]{1}[0-9]{1}|3[0-1]{1}))?)?' for type 'DatumMetOnzekerheid-s'.